addon/maptools/rectangletool.js

```diff
@@ -3,31 +3,35 @@ import MapTool from 'ember-flexberry-gis/maptools/maptool';
 export default MapTool.extend({
 
   _layer: null,
+  _measure: null,
 
-  rectangleStarted({ layer }) {
```

Comment:
this code was used in other tools - zoom, zoom out, and identify. You should restore it.